lib/guardian/token/jwt.ex

```diff
@@ -311,8 +310,10 @@ defmodule Guardian.Token.Jwt do
   end
 
   defp fetch_secret(mod, opts) do
-    (opts |> Keyword.get(:secret) |> Config.resolve_value() || apply(mod, :config, [:secret_key]))
-    |> case do
+    secret = Keyword.get(opts, :secret)
```

Comment:
I found this pipe to be very difficult to follow especially since it pipes into a `case`.  I think breaking this out makes the code easier to grok.